features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/locales/jaggery/locale_default.json

```diff
@@ -650,6 +656,11 @@
     "Error while updating API swagger definition" : "Error while updating API swagger definition",
     "Delete Task" : "Delete Task",
     "Authorization Header": "Authorization Header",
+    "API Security": "API Security",
     "Here a custom authorization header can be defined as a replacement to the default <strong>Authorization</strong> header used to send a request. If a value is specified here, that will be used as the header field to send the access token in a request to consume the API": "A custom authorization header can be defined here as a replacement to the default <strong>Authorization</strong> header used to send a request. If a value is specified here, it will be used as the header field to send the access token in a request to consume the API",
-    "You have selected Microgateways for this API. The mediation policies selected here will not be applied in Microgateway environments.":"You have selected Microgateways for this API. The mediation policies selected here will not be applied in Microgateway environments."
+    "You have selected Microgateways for this API. The mediation policies selected here will not be applied in Microgateway environments.":"You have selected Microgateways for this API. The mediation policies selected here will not be applied in Microgateway environments.",
+    "This option determines the type of security that will be used to secure this API. An API can be secured with either one of them or it can be secured by both of them. If OAuth2 option is selected, relevant API will require a valid OAuth2 token for successful invocation. If Mutual SSL option is selected, a trusted client certificate should be presented to access the API":"This option determines the type of security that will be used to secure this API. An API can be secured with either one of them or it can be secured by both of them. If OAuth2 option is selected, relevant API will require a valid OAuth2 token for successful invocation. If Mutual SSL option is selected, a trusted client certificate should be presented to access the API",
```

Comment:
Could you tell me where this message is used so that I can help you rephrase this to bring out the meaning better? Can you attach a UI screen shot?

Reply:
This message is used as the help text, when the user is trying the select the API security.
![screenshot from 2018-10-12 17-10-54](https://user-images.githubusercontent.com/4003116/46867259-ff867c80-ce41-11e8-8678-0b5c26467553.png)